src/azure-cli/azure/cli/command_modules/cdn/_help.py

```diff
@@ -987,3 +987,658 @@
+    type: string
+    short-summary: >
+        Type of health probe request. Allowed to be GET and HEAD.
+  - name: --probe-protocol
+    type: string
+    short-summary: >
+        Protocol to use for health probe. Allowed to be Http and Https.
+  - name: --probe-interval-in-seconds
+    type: int
+    short-summary: >
+        The number of seconds between health probes. Default is 240 seconds.
+  - name: --probe-path
+    type: string
+    short-summary: >
+        The path relative to the origin that is used to determine the health of the origin.
+  - name: --load-balancing-sample-size
+    type: int
+    short-summary: >
+        The number of samples to consider for load balancing decisions.
+  - name: --load-balancing-successful-samples-required
+    type: int
+    short-summary: >
+        The number of samples within the sample period that must succeed for the orgin to be deemed as healthy.
+  - name: --load-balancing-additional-latency-in-milliseconds
+    type: int
+    short-summary: >
+        The additional latency in milliseconds for probes to fall into the lowest latency bucket.
+short-summary: Creates a new origin group within the specified profile.
+examples:
+  - name: Creates a new origin group within the specified profile.
+    text: >
+        az afd origin-group create -g group --origin-group-name og1 --profile-name profile
+        --probe-request-type GET --probe-protocol Http --probe-interval-in-seconds 120 --probe-path /test1/azure.txt
+        --load-balancing-sample-size 4 --load-balancing-successful-samples-required 3
+        --load-balancing-additional-latency-in-milliseconds 50
+"""
+
+helps['afd origin-group update'] = """
+type: command
+parameters:
+  - name: --origin-group-name
+    type: string
+    short-summary: >
+        The Name of the origin group.
+  - name: --probe-request-type
+    type: string
+    short-summary: >
+        Type of health probe request. Allowed to be GET and HEAD.
+  - name: --probe-protocol
+    type: string
+    short-summary: >
+        Protocol to use for health probe. Allowed to be Http and Https.
+  - name: --probe-interval-in-seconds
+    type: int
+    short-summary: >
+        The number of seconds between health probes. Default is 240 seconds.
+  - name: --probe-path
+    type: string
+    short-summary: >
+        The path relative to the origin that is used to determine the health of the origin.
+  - name: --load-balancing-sample-size
+    type: int
+    short-summary: >
+        The number of samples to consider for load balancing decisions.
+  - name: --load-balancing-successful-samples-required
+    type: int
+    short-summary: >
+        The number of samples within the sample period that must succeed for the orgin to be deemed as healthy.
+  - name: --load-balancing-additional-latency-in-milliseconds
+    type: int
+    short-summary: >
+        The additional latency in milliseconds for probes to fall into the lowest latency bucket.
```

Comment:
The help message of parameter can be defined in _params.py.

Reply:
I found you already setting the help property of parameter in _params.py. Then you don't need to write them again here